packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -627,6 +627,27 @@ export class DataStores implements IDisposable {
         }
     }
 
+    /**
+     * When running GC in test mode, this is called to delete objects whose routes are unused. This enables testing
+     * scenarios with accessing deleted content.
+     * @param unusedRoutes - The routes that are unused in all data stores in this Container.
+     */
+    public tombstoneUnusedRoutes(unusedRoutes: string[]) {
+        for (const route of unusedRoutes) {
+            const pathParts = route.split("/");
+            // Delete data store only if its route (/datastoreId) is in unusedRoutes. We don't want to delete a data
+            // store based on its DDS being unused.
+            if (pathParts.length > 2) {
+                continue;
+            }
+            const dataStoreId = pathParts[1];
```

Comment:
What about attachment blobs? How will they be handled after your changes?

Reply:
After some thought, I don't think we need to tombstone attachment blobs. It would just be wise to delete them.